command/top.go

```diff
@@ -10,26 +11,55 @@ import (
 )
 
 const (
-	topHelp = "Displays the most used distro, terminal, etc."
+	topHelp  = "Displays the most used distro, terminal, etc."
+	topUsage = "`!top` or `!top [Distro OR DeWm OR Terminal etc.]`"
 )
 
 func top(ctx *Context, args []string) {
-	topFields, err := db.TopSysinfoFields()
-	if err != nil {
-		ctx.ReportError("Failed to get top fields", err)
-		return
-	}
+	if len(args) == 2 {
```

Comment:
contents of this if (or else) could be moved to a separate function, like `topAll`, `topSpecific`

Reply:
what do you mean here?